### ISSUE TITLE
 ERROR: Attribute 'assigned_object_type' for '10.33.33.1/24' invalid: ipam.fhrpgroup

### ISSUE THREAD

**Issue by sliddjur** [bb-Ricardo/netbox-sync#137]
If an this object (IPAM > Other > FHRP Group) exists in netbox, the sync will fail with this error message:

```
2022-01-31 12:39:39,301 - INFO: Finished querying necessary objects from NetBox
2022-01-31 12:39:39,361 - ERROR: Attribute 'assigned_object_type' for '10.33.33.1/24' invalid: ipam.fhrpgroup
Error while resolving relations for 10.33.33.1/24
```

**Comment by bb-Ricardo** [comment]
Hi, Thank you. This is interesting. Unfortunately the OpenAPI docs don't specify all available values 🙄.

It seems the IP iss assigned to the FHRP group. Is that correct?

**Comment by sliddjur** [comment]
@bb-Ricardo yes that is correct!

**Comment by bb-Ricardo** [comment]
Hey @sliddjur can you check out latest development branch and test if it works with that one?

**Comment by bb-Ricardo** [comment]
This should fix it. There are only 3 options for 'assigned_object_type'

<img width="785" alt="Bildschirmfoto 2022-01-31 um 15 16 41" src="https://user-images.githubusercontent.com/771836/151809566-75c204a3-c333-4ac1-a897-5b85f5328526.png">

**Comment by bb-Ricardo** [comment]
fixed in version 1.2.3
